### ISSUE TITLE
Don't remind about model update

### ISSUE THREAD

**Issue by Kombiusz** [LykosAI/StabilityMatrix#1069]
### Proposal

**Proposition** 

Add option to right click to "Ignore this Update" or some other way to say it, I'm asking for it because there are multiple LORAs im using that have multiple versions depending on what is base type for example there is version for SD 1.5 and Pony but I wanted only 1.5 version, and yet there is still "update" hanging because Matrix is detecting that there is new version aka Pony, so I'm hoping that there would be an option to "ignore" update until new version shows up

tldr; I want right click option to ignore "update available" until next update shows up
